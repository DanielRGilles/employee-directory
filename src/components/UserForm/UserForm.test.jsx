import {render } from  '@testing-library/react';
import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min';
import { UserProvider } from '../../context/UserContext';

import UserForm from './UserForm';

jest.mock('../../context/UserContext')
jest.mock('../../services/users.js')

describe('User form component test', () => {
  
    it('renders the form', () => {
       const { container } = render(
            <MemoryRouter>
                <UserProvider>
                     <UserForm/>
                </UserProvider>
            </MemoryRouter>
        );
        expect(container).toMatchSnapshot();
         
    })

})