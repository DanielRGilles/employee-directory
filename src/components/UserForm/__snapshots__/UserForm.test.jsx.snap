// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User form component test renders the form 1`] = `
<div>
  <form
    class=""
  >
    <fieldset
      class="form"
    >
      <legend>
        Authenticate
      </legend>
      <section
        class="formSection"
      >
        <label
          for="email"
        >
          Email
        </label>
        <input
          class="input"
          id="email"
          name="email"
          type="email"
          value=""
        />
      </section>
      <section
        class="formSection"
      >
        <label
          for="password"
        >
          Password
        </label>
        <input
          class="input"
          id="password"
          name="password"
          type="password"
          value=""
        />
      </section>
      <button
        type="submit"
      >
        Authenticate
      </button>
      
    </fieldset>
  </form>
</div>
`;
